src/cli.ts

```diff
@@ -15,87 +16,162 @@ const defaultCommand = define({
       type: 'string',
       default: 'auto',
       description: `Package manager to use for packing ('auto' | 'npm' | 'yarn' | 'pnpm' | 'bun')`
+    },
+    'list-tarball-files': {
+      type: 'boolean',
+      default: false,
+      description: 'List all files in the tarball',
     }
   },
   async run(ctx) {
     const root = ctx.positionals[0];
     let pack = ctx.values.pack as PackType;
+    const showAllFiles = ctx.values['list-tarball-files'] as boolean;
```

Comment:
i wonder if this should actually just be a debug level log output

i.e. if `--debug` or something is passed, it dumps the tgz listing

Reply:
i can convert it, but dunno if you're thinking of removing the current flag and add `--debug` flag and only show the tarball listing when this new flag is passed. Let me know your thoughts tho before moving forward.

Reply:
I think we should have a `--debug` or `--log-level=debug` flag and that enables this (and other things)